common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -164,27 +164,34 @@ private object Logging {
     if (simpleName.endsWith("$")) simpleName.dropRight(1)
     else simpleName
   }
-
 }
 
 private object Emitter {
   val timeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").withZone(ZoneId.of("UTC"))
 }
 
-case class LogMarkerToken(component: String, action: String, state: String) {
+case class LogMarkerToken(component: String,
```

Comment:
A ScalaDoc comment here would be great, stating that `macroTags` should be used for tags with a bounded number of permutations only while microTags can be used for whatever granularity you might need.